cleanlab/filter.py

```diff
@@ -290,38 +281,43 @@ def get_noise_indices(
 
     num_to_remove_per_class : list of int of length K (# of classes)
       e.g. if K = 3, num_to_remove_per_class = [5, 0, 1] would return
-      the indices of the 5 most likely mislabeled examples in class s = 0,
-      and the most likely mislabeled example in class s = 1.
+      the indices of the 5 most likely mislabeled examples in class labels = 0,
+      and the most likely mislabeled example in class labels = 1.
 
       Note
       ----
-      Only set this parameter if ``prune_method == 'prune_by_class'``
-      You may use with ``prune_method == 'prune_by_noise_rate'``, but
+      Only set this parameter if ``filter_by == 'prune_by_class'``
+      You may use with ``filter_by == 'prune_by_noise_rate'``, but
       if ``num_to_remove_per_class == k``, then either k-1, k, or k+1
       examples may be removed for any class. This is because noise rates
       are floats, and rounding may cause a one-off. If you need exactly
       'k' examples removed from every class, you should use ``'prune_by_class'``
 
-    prune_method : str (default: 'prune_by_noise_rate')
-      Possible Values: 'prune_by_class', 'prune_by_noise_rate', or 'both'.
-      Method used for pruning.
+    filter_by : str (default: 'prune_by_noise_rate')  TODO: change default to cl_off_diag?
+      Possible Values: {'prune_by_class', 'prune_by_noise_rate', 'both',
```

Comment:
why include `prune_by_*` prefix when the arg name already says `filter_by`?

Reply:
convention. methods are called PBNR, PBC in the paper (PB = prune by). agree its redundant, but didn't feel it was necessary to further separate the paper naming of the method (where many of our developers come from) and potentially confuse folks as to what the method means over 8 chars.  (e.g. `prune_by_class` would become `class` which is a terrible name for a method)